server/history/util.py

```diff
@@ -20,8 +20,9 @@
 
 
 def get_history_db_connection(with_db_name=True):
+
     dbname = secrets.POSTGRES_DB if with_db_name else None
     if secrets.POSTGRES_PASS == '':
-        return psycopg2.connect(user=secrets.POSTGRES_USER, dbname=dbname)
+        return psycopg2.connect(user=secrets.POSTGRES_USER, dbname=dbname, host="postgres", port="5432")
```

Comment:
is this file used in the actual deployment at all? @mathcolo

Reply:
I believe so—db related stuff should really be in one file huh

Reply:
I just want to make sure that if this is used by the real deployment that Im not hardcoding this host value strictly for docker usage.

Reply:
Gotcha. Yep!